data/xp-pen-artist12.tablet

```diff
@@ -0,0 +1,29 @@
+# XP-Pen
+# Artist 12
+#
```

Comment:
Only think I can see missing here: can you add the sysinfo information here please. Please see https://github.com/linuxwacom/wacom-hid-descriptors/ and existing files for guidance, e.g. 9d61a6c0cf4c0052a53afa288a4bf900e4eb2018

Reply:
Is this it? [sysinfo.LbiyIAU6IY.tar.gz](https://github.com/linuxwacom/libwacom/files/9081759/sysinfo.LbiyIAU6IY.tar.gz)

Reply:
open an issue at the wacom-hid-descriptors repo please and attach the file there, then add the link to that issue here. This way it can be merged and they're all in the same place.

Reply:
Here it is https://github.com/linuxwacom/wacom-hid-descriptors/issues/238